locales/nn-NO/src/structure.ts

```diff
@@ -104,6 +106,11 @@ export default removeUndefinedLocaleResources({
   'banners.reference-changed-banner.reason-removed.text':
     'Denne referansen har blitt fjerna sidan du opna den.',
 
+  /** Browser/tab title when creating a new document of a given type */
+  'browser-document-title.new-document': 'Nytt {{schemaType}}',
```

Comment:
This is a bit tricky when the adjective "ny" is conjugated based on the noun 🤔

Reply:
Agree. For now we do not translate schema type names either, which makes this doubly hard. We'll think about it some more once we start localizing them, but for now I think the question is whether to go with "Ny" or "Nytt" 🤔